### ISSUE TITLE
Feature Request: Support Full-finetuning of Qwen-Image on RTX PRO 6000 Max-Q (96GB VRAM)

### ISSUE THREAD

**Issue by alfredplpl** [kohya-ss/musubi-tuner#490]
Summary

I’d like to formally request support in Musubi Tuner for full-parameter finetuning of the Qwen-Image diffusion model (beyond LoRA).
With GPUs like the RTX PRO 6000 Max-Q (96GB VRAM), it is now practical to perform full finetuning on a single device, and adding first-class support for this would make Musubi Tuner more versatile for advanced use cases.

Why this matters
	•	Qwen-Image is increasingly used for tasks such as high-quality text rendering, fine-grained editing, and domain-specific adaptation.
	•	While LoRA is lightweight and efficient, certain scenarios (e.g., brand alignment, large-scale domain transfer, instruction-following fidelity) require full finetuning.
	•	Many researchers and practitioners now have access to workstation-class GPUs with 96GB VRAM, removing the main bottleneck that previously made full finetuning impractical.

Requested feature
	•	Provide official support (scripts/configs) for end-to-end full finetuning of Qwen-Image in Musubi Tuner.
	•	Ensure compatibility with Musubi Tuner’s inference and evaluation workflows.
	•	Document recommended training settings (batch sizes, precision modes, checkpointing options) for 96GB-class GPUs.

Target hardware
	•	GPU: RTX PRO 6000 Max-Q (96GB VRAM)
	•	OS/stack (planned): Ubuntu 22.04, CUDA 12.x, PyTorch ≥ 2.3

What success looks like
	•	Users can run full finetuning of Qwen-Image directly with Musubi Tuner on 96GB GPUs.
	•	Generated checkpoints integrate seamlessly with existing Musubi Tuner evaluation and inference tools.
	•	Clear documentation or example configs for full finetuning are available in the repository.

Happy to contribute

I’m happy to test training runs on RTX PRO 6000 Max-Q, benchmark VRAM/throughput, and contribute documentation or example configs to help land this feature.

**Comment by kohya-ss** [comment]
Thank you for your excellent and detailed feature request!
We agree that full finetuning support for Qwen-Image would be a very valuable addition to Musubi Tuner, especially for advanced use cases where LoRA has its limits. We will positively consider implementing this feature.

We also truly appreciate your offer to help with testing and documentation. That's a great help for us.
Once we create a pull request for this feature, we would be grateful for your cooperation in reviewing it and testing on your hardware.
We will keep you updated on our progress in this issue.

**Comment by sdbds** [comment]
Honestly, the main problem is that the LoRA architecture is gradually falling behind people's needs.....
I think we could also try more new architectures, such as T-LoRA, etc....
https://arxiv.org/abs/2507.05964v1

**Comment by alfredplpl** [comment]
I got the GPU.
<img width="1290" height="562" alt="Image" src="https://github.com/user-attachments/assets/98070a64-35c2-46a1-bd82-46f3a932e519" />

**Comment by alfredplpl** [comment]
Comfirm that it works! 🎉

<img width="645" height="292" alt="Image" src="https://github.com/user-attachments/assets/ca64ca6e-1dc7-46e9-beab-68b2fd763197" />

<img width="1240" height="55" alt="Image" src="https://github.com/user-attachments/assets/806e9225-8b17-4efc-a41a-e02b125f3434" />

**Comment by kohya-ss** [comment]
> Comfirm that it works! 🎉

Thank you for the confirmation! If the training results are fine, I'd like to merge this PR.

**Comment by kohya-ss** [comment]
> Honestly, the main problem is that the LoRA architecture is gradually falling behind people's needs.....
> I think we could also try more new architectures, such as T-LoRA, etc....
> https://arxiv.org/abs/2507.05964v1

Thank you for sharing this.

I think it's possible to implement a new LoRA or something, but I feel the issue will always be the inference implementation...

**Comment by alfredplpl** [comment]
I generated 2 images by the original model and a fullfinetuned model in term of Japanese font rendering.

Original:
![image](https://github.com/user-attachments/assets/b48bea61-e9a7-44e2-b472-050dc69476b3)

Fullfinetuned (WIP) :
![image](https://github.com/user-attachments/assets/bf5098ff-9948-4d1e-b45f-8dc2b9cb17a5)

Note: same prompt, same seed
